nemo/collections/asr/parts/activations.py

```diff
@@ -18,10 +18,8 @@
 __all__ = ['Swish']
 
 
-class Swish(nn.Module):
+class Swish(nn.SiLU):
     """
     Swish activation function introduced in 'https://arxiv.org/abs/1710.05941'
+    Mathematically identical to SiLU. See note in nn.SiLU for references.
```

Comment:
There is an "import torch" on the top which is not used.

Reply:
Fixed